### PR TITLE
[ListCard / SimpleList] bug fixes

### LINKED ISSUE #1689: [SimpleList] pagination can't be hidden
### What package is this for?

- [x] React
- [ ] Angular

### Describe the bug

There is currently no way to hide the pagination, even if theres only 1 page of data

### Expected behavior

Should be able to hide the pagination

### LINKED ISSUE #1690: [SimpleList] i18n is not being respected
### What package is this for?

- [x] React
- [ ] Angular

### Describe the bug

i18n prop should be merged with defaultProps so the user doesn't have to provide all i18n

### To Reproduce

1. Go to any List story
2. add 1 of the 18n as a prop
3. See that errors are thrown because the defaultProps of i18n are not being respected

### Expected behavior

Consumer should be able to provide only some of the i18n and let the defaultprops take care of the rest

### LINKED ISSUE #1691: [ListCard] add empty state
### What package is this for?

- [x] React
- [ ] Angular

### Describe the bug

There's currently no empty state for the ListCard

### Expected behavior

Should have an empty state if data has not loaded in yet

### PR DESCRIPTION
**Pull request by sstone2423**
Closes #1689
Closes #1690 
Closes #1691 

**Summary**

- Adds empty state for ListCard if there is no data
- Merges `i18n` prop with defaultProps `i18n`
- Adds ability to hide the pagination in SimpleList

**Change List (commits, features, bugs, etc)**

- Added story for empty state ListCard
- Added story for hidden pagination in SimpleList

**Acceptance Test (how to verify the PR)**

- Go to ListCard - empty story to see empty state
- Go to SimpleList - hidden pagination story to see hidden pagination AND there is only 1 `i18n` prop being passed here so this also verified the merged i18n

### CODE REVIEW

**Comment by JoelArmendariz** [comment]
Looks like the empty state `ListCard` is ignoring width/card-size and is stretching across the container when empty.

Screen Shot 2020-10-15 at 12 10 53 PM

**Comment by sstone2423** [comment]
> Looks like the empty state `ListCard` is ignoring width/card-size and is stretching across the container when empty.
> 
> Screen Shot 2020-10-15 at 12 10 53 PM

Fixed by add a width wrapper around the ListCard as we do in other Card stories

**Review by JoelArmendariz — APPROVED** [review]
LGTM

**Review by scottdickerson — APPROVED** [review]
LGTM

File: src/components/List/SimpleList/SimpleList.jsx [hunk 1]
```diff
@@ -51,6 +51,12 @@ const propTypes = {
   pageSize: PropTypes.string,
   /** callback function returned a modified list */
   onListUpdated: PropTypes.func,
+  /** optionally renders SimplePagination at the bottom of the list */
+  showPagination: PropTypes.bool,
```
Comment (tay1orjones) [inline src/components/List/SimpleList/SimpleList.jsx:55]: In HierarchyList, Table, and a few other components this is typically named `hasPagination`. 
```suggestion
  hasPagination: PropTypes.bool,
```

**Review by tay1orjones — CHANGES_REQUESTED** [review]
Just one small change for the sake of consistency. Thanks for removing all those empty `info` blocks!

**Review by tay1orjones — APPROVED** [review]
I went ahead and pushed an update to your branch to rename the prop 👍

**Pull request merged**